pkg/flare/archive_win.go

```diff
@@ -100,6 +100,28 @@ func zipTypeperfData(tempDir, hostname string) error {
 	}
 	return nil
 }
+func zipLodctrOutput(tempDir, hostname string) error {
+	cmd := exec.Command("lodctr", "/q")
```

Comment:
Should we put a timeout on this ? Or are we sure it will never block ?

Reply:
I added a timeout on this and the other function.